content/docs/guide.md

```diff
@@ -99,54 +280,41 @@ template by running the `init` command:
 # See the documentation for more information:
 # https://packit.dev/docs/configuration/
 
-specfile_path: packit.dev.spec
+specfile_path: directory.spec
 
 # add or remove files that should be synced
 files_to_sync:
-    - packit.dev.spec
+    - directory.spec
     - .packit.yaml
 
 # name in upstream package repository/registry (e.g. in PyPI)
-upstream_package_name: packit.dev
+upstream_package_name: directory
 # downstream (Fedora) RPM package name
-downstream_package_name: packit.dev
+downstream_package_name: directory
 ```
 
 Head on to page [configuration]({{< ref "configuration" >}}) to learn more
 about the config file.
 
-Packit supports actions and hooks: you can define your own commands which would
-replace packit's implementation (e.g. get version, or create a tarball) or run
-specific commands after the upstream repo is cloned. Please read more about
-actions here: [actions]({{< ref "actions" >}}).
-
 Packit needs an RPM spec file to build your package. The spec file does not
 need to be included in the upstream repo and can be generated on the fly or
-downloaded (e.g. from Fedora dist-git). Please check out our [FAQ]({{< ref
-"faq" >}}) to read about some other common questions.
-
-You can also read more about the SRPM process in [the document]({{< ref
-"cli/srpm" >}}) dedicated to the `srpm` command.
+downloaded (e.g. from Fedora dist-git).
+Please check out our [FAQ]({{< ref "faq" >}}) to read about some other common questions.
 
-Is your `packit srpm` finally passing? If not, feel free to reach out to us. If
-yes, let's proceed to the next level.
+You can also read more about the SRPM process
+in [the document]({{< ref "cli/srpm" >}}) dedicated to the `srpm` command.
 
-
-## Running from the dist-git repository
+##### Running from the dist-git repository
 
 When you are not allowed or do not want to run packit command from the upstream
 git repository, you can run commands from the cloned dist-git repository in the
 same way as you do from upstream.  Make sure, that you specify the
 `upstream_project_url` in your configuration.
 
-## You are approved to use Packit Service.
-
-In order to start using packit service, our GitHub app, you need to [install it
-in your GitHub projects]({{< ref "packit-service" >}}) and get approved.
-(The approval process can be done by you without any manual interaction
-by the Packit team.)
+Is your `packit srpm` finally passing? If not, feel free to reach out to us.
+If yes, let's proceed to the next level.
 
-## The project has successful builds inside the service.
+#### The project has successful builds inside the service
 
 If a SRPM can be created locally, all should be good in the service as well.
 That's the theory. In practice, your laptop and packit service sandbox
```

Comment:
do we want to still mention the sandbox in the first place (mainly in relation to packit/packit-service#1583)

Reply:
I would change it once we change the behaviour for new users.